### FIELD DATA CAN BE ONLY ONE OF THE TEXT BUTTON OR IMAGE BUTTON ###
# A button with text and onclick action
string text_button_name 
google_chat_ros/OnClick text_button_on_click
# An image button with an onclick action
string image_button_name
google_chat_ros/OnClick image_button_on_click
string icon # see https://developers.google.com/chat/api/reference/rest/v1/cards#icon
string original_icon_url # Original icon image's url
string original_icon_filepath # Original icon
